python/PyQt6/core/class_map.yaml

```
@@ -1228,15 +1228,19 @@ QgsArchive: src/core/qgsarchive.h#L35
 QgsArrowInferSchemaOptions.geometryColumnName: src/core/qgsarrowiterator.h#L123
 QgsArrowInferSchemaOptions.setGeometryColumnName: src/core/qgsarrowiterator.h#L115
 QgsArrowInferSchemaOptions: src/core/qgsarrowiterator.h#L103
-QgsArrowIterator.QgsArrowIterator: src/core/qgsarrowiterator.h#L272
-QgsArrowIterator.inferSchema: src/core/qgsarrowiterator.h#L296
-QgsArrowIterator.inferSchema: src/core/qgsarrowiterator.h#L303
-QgsArrowIterator.nextFeatures: src/core/qgsarrowiterator.h#L289
-QgsArrowIterator.setSchema: src/core/qgsarrowiterator.h#L279
-QgsArrowIterator: src/core/qgsarrowiterator.h#L265
+QgsArrowIterator.QgsArrowIterator: src/core/qgsarrowiterator.h#L332
+QgsArrowIterator.inferSchema: src/core/qgsarrowiterator.h#L364
+QgsArrowIterator.inferSchema: src/core/qgsarrowiterator.h#L371
```

YAMLlint (1.37.1)

[error] duplication of key "QgsArrowIterator.inferSchema" in mapping

(key-duplicates)